### DISCUSSION TITLE
Why does debug button disappear when minimize stage?

### DISCUSSION THREAD

**Discussion by itta611** [ScratchAddons/ScratchAddons#3516] [Help] [answered]
When you change the size of the stage, the debug button disappears.
I watched the code of debugger, it seemed to do so on purpose.
https://github.com/ScratchAddons/ScratchAddons/blob/1076940637e68ed8550587ea6585d38693dfa12e/addons/debugger/style.css#L10

**Comment by mxmou** [comment, marked as answer]
There isn't any space for it if small stage and turbo mode is enabled.

**Comment by itta611** [comment]
![スクリーンショット 2021-09-19 午前7 58 06](https://user-images.githubusercontent.com/86896657/133910596-c6f7c11e-be84-4ef0-98c7-66cfd466779d.png)
I don't think so...

**Reply by Samq64** [reply]
With turbo mode enabled.
